urls.py

```diff
@@ -10,6 +10,7 @@
 from django.contrib.staticfiles.urls import staticfiles_urlpatterns
 from django.contrib.sitemaps import views as sitemap_views
 from django.utils import cache
+from django.views.decorators.cache import cache_page
```

Comment:
This is a fairly substantial functionality change: 

https://docs.djangoproject.com/en/2.0/topics/cache/#django.views.decorators.cache.cache_page

It means that your pages will be cached using the default cache defined in your Django settings as well as by Varnish. That will require new cache purging logic as well as introducing a second source of performance and space usage concerns.

Django 1.10 added a standard `cache_control` decorator which appears to be a direct replacement for the poorly-named `cache_page` decorator which is currently in this file:

https://docs.djangoproject.com/en/2.0/topics/http/decorators/#django.views.decorators.cache.cache_control

Reply:
unfortunately we are using django 1.8 so we can't use `cached_control`

Reply:
In that case I'd leave the working code in here and add a comment that it can be replaced when we upgrade to Django 1.10+